test/ex_doc/formatter/html_test.exs

```diff
@@ -5,10 +5,10 @@ defmodule ExDoc.Formatter.HTMLTest do
 
   setup_all do
     File.mkdir(output_dir)
-    File.copy("test/fixtures/README.md", "test/README.md")
+    File.copy("test/fixtures/README.md", "test/tmp/README.md")
 
     on_exit fn ->
-      File.rm("test/README.md")
+      File.rm("test/tmp/README.md")
       System.cmd "rm", ["-rf", "#{output_dir}"]
```

Comment:
I'm  also in favour of getting rid of these two lines, so we can actually see the output when debugging exdoc.
Anyways they will be removed on `mix clean`